### PR TITLE
Hero mixin

### PR DESCRIPTION
**Pull request by RazvanDH**
updates to the hero component according to the design specs

### CODE REVIEW

**Comment by igoratron** [comment]
**question** why was 8e5a36d necessary? It merges the branch into itself

File: src/sass/_mixins/hero.scss [hunk 1]
```diff
@@ -227,3 +229,31 @@
...
 }
+
+
+
+/* ---------------------------------------------------------------------- *\
+  SELECTORS FOR TITLE / SUBTITLE / CONTENT
+  - because we apply the same selectors on multiple resolutions, I've stored them inside a mixin for easier use / update
+\* ---------------------------------------------------------------------- */
+
+@mixin heroTitle {
```
Comment (igoratron) [inline src/sass/_mixins/hero.scss]: **niptick** this could be a CSS class

File: src/sass/general/hero.scss [hunk 2]
```diff
@@ -32,31 +31,37 @@
...
-      margin-bottom: 0;
+    @include heroSubtitle {
+      @include font(heroFontNote($heroFontNoteSubtitle, small), true, 400);
+      margin-top: rem(.5);
     }
 
-    // small size
+    @include heroContent {
+      display: block;
+      margin: rem(2) 0 0;
```
Comment (igoratron) [inline src/sass/general/hero.scss]: **nitpick** maybe `rem(2 0 0)`

**Comment by RazvanDH** [comment]
@igoratron - got rid of the weird commit

File: src/sass/_mixins/hero.scss (hunk 1 above)
Comment (RazvanDH) [inline src/sass/_mixins/hero.scss]: fixed. we now have classes for that

File: src/sass/general/hero.scss (hunk 2 above)
Comment (RazvanDH) [inline src/sass/general/hero.scss]: fixed

**Comment by igoratron** [comment]
**LGTM**

**Pull request merged**